Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM jboss/wildfly:8.2.0.Final
 MAINTAINER Democracy Works, Inc. <dev@democracy.works>
 
+ENV HOST_IP "TO.BE.FILLED.IN"
+
 COPY ./start /bin/start
 
+ADD standalone-rsyslog.xml /opt/jboss/wildfly/standalone/configuration/standalone-rsyslog.xml
+
 ENTRYPOINT ["/bin/start"]
-CMD ["-b", "0.0.0.0", "-bmanagement", "0.0.0.0"]
+CMD ["-c", "standalone-rsyslog.xml", "-Djboss.host.name=$HOST_IP", "-b", "0.0.0.0", "-bmanagement", "0.0.0.0"]
```

Comment:
What does jboss.host.name default to? If it's the container's hostname, we could just use that, drop the ENV on line 4, and set the hostname in the `docker run` command (which can even be the same as the CoreOS node's; other containers are already doing this).